utils/api/postcodeAPI.ts

```diff
@@ -24,7 +24,14 @@ export const normalizeAddress = (address: AddressAPI): Address => {
   };
 };
 
-export const lookupPostcode = async (postcode: string): Promise<Address[]> => {
-  const { data } = await axios.get(`/api/postcode/${postcode}`);
-  return data.address.map(normalizeAddress);
+export const lookupPostcode = async (
+  postcode: string,
+  page_number = 1,
+  building_number?: string
+): Promise<AddressWrapper> => {
+  const { data } = await axios.get(
+    `/api/postcode/${postcode}?page=${page_number}&buildingNumber=${building_number}`
```

Comment:
Could we add a test for this change? See https://github.com/LBHackney-IT/lbh-social-care-frontend/blob/a582468826dcc24ee9880d31fc50cfcf5b9e688e/utils/api/relationships.spec.ts#L149-L160 for an example.

Reply:
Test added which required some changes in the implementation code to cover response being undefined.